rust/agents/relayer/src/settings/matching_list.rs

```diff
@@ -178,19 +178,23 @@ impl<'de> Deserialize<'de> for Filter<H256> {
 #[derive(Debug, Deserialize, Clone)]
 #[serde(tag = "type")]
 struct ListElement {
-    #[serde(default, rename = "sourceDomain")]
-    src_domain: Filter<u32>,
-    #[serde(default, rename = "sourceAddress")]
-    src_address: Filter<H256>,
-    #[serde(default, rename = "destinationDomain")]
-    dst_domain: Filter<u32>,
-    #[serde(default, rename = "destinationAddress")]
-    dst_address: Filter<H256>,
+    #[serde(default, rename = "senderDomain")]
```

Comment:
origin domain

Reply:
I just want to confirm that we want to have

`originDomain`, `destinationDoimain`
`senderAddress`, `recipientAddress`

this feels inconsistent to me.